core/src/test/java/org/mskcc/cbio/oncokb/util/IndicatorUtilsTest.java

```diff
@@ -103,14 +105,49 @@ public void testProcessQuery() throws Exception {
         assertTrue("The treatments are not the same.", resp1.getTreatments().equals(resp2.getTreatments()));
 
         // Check unknown denominator fusion, it should return same data as querying specific fusion.
-        query1 = new Query("BRAF", null, "CUL1-BRAF Fusion", null, "Ovarian Cancer", null, null, null);
-        query2 = new Query("CUL1-BRAF", null, null, "fusion", "Ovarian Cancer", null, null, null);
+        query1 = new Query(null, null, null, "BRAF", "CUL1-BRAF Fusion", null, "Ovarian Cancer", null, null, null);
+        query2 = new Query(null, null, null, "CUL1-BRAF", null, "fusion", "Ovarian Cancer", null, null, null);
+        resp1 = IndicatorUtils.processQuery(query1, null, null, null, true);
+        resp2 = IndicatorUtils.processQuery(query2, null, null, null, true);
+        assertTrue("Oncogenicities are not the same", resp1.getOncogenic().equals(resp2.getOncogenic()));
+        assertTrue("Treatments are not the same", resp1.getTreatments().equals(resp2.getTreatments()));
+        assertTrue("Highest sensitive levels are not the same.", LevelUtils.areSameLevels(resp1.getHighestSensitiveLevel(), resp2.getHighestSensitiveLevel()));
+        assertTrue("Highest resistance levels are not the same.", LevelUtils.areSameLevels(resp1.getHighestResistanceLevel(), resp2.getHighestResistanceLevel()));
+
+        // Check hugoSymbol and entrezGene pair
+        query1 = new Query(null, null, 673, null, "V600E", null, "Melanoma", null, null, null);
+        query2 = new Query(null, null, null, "BRAF", "V600E", null, "Melanoma", null, null, null);
+        resp1 = IndicatorUtils.processQuery(query1, null, null, null, true);
+        resp2 = IndicatorUtils.processQuery(query2, null, null, null, true);
+        assertTrue("Genes are the same", resp1.getGeneSummary().equals(resp2.getGeneSummary()));
```

Comment:
The message should be "Genes are not the same", right? same with 2 other assertTrue() below.